src/background/chrome-api.js

```diff
@@ -72,19 +74,19 @@ export function getChromeAPI(chrome = globalThis.chrome) {
    * @return {(...args: Args) => Promise<Result>}
    */
   const promisifyAlt = fn => {
-    // @ts-ignore
+    // @ts-expect-error
     return promisify(fn);
   };
```

Comment:
Not related to this PR, but I just remembered I wanted to ask about this.

What's the purpose of `promisifyAlt`? Isn't it just forwarding to `promisify`? Would it be the same to just use `promisify`, or am I missing something?

Reply:
This is a typing hack that gives `promisify` a different signature. It avoids needing to mark every place where `promisifyAlt` is used as an expected error.

Reply:
ooohhh, I see, Isee. I didn't realize the jsdoc was different on them.